ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -44,14 +46,23 @@ const AdvancedGasFeeDefaults = () => {
     if (isDefaultSettingsSelected) {
       dispatch(setAdvancedGasFee(null));
       setDefaultSettingsSelected(false);
+      updateTransactionEventFragment({
+        advanced_gas_defaults_updated: true,
```

Comment:
You need to specify the properties/sensitiveProperies subkeys otherwise these get written to the fragment options instead of the payload that gets sent to segment.

Reply:
sensitiveProperties are reserved for things that will show up on chain,

Reply:
Hey @brad-decker : in case sensitiveProperties is reserved for on chain values, should I updated [this](https://github.com/MetaMask/metamask-extension/pull/13385/files#diff-675c437262d906486cf5436c0b64535347286cc4d1081dbd863fb45b8a01b81aR1513) PR to use properties ?

Reply:
@jpuri I think we should keep these where they are because we'll want to compare them to actual values used (on chain) and they don't serve much purpose on the non-anonymized events. so no, lets keep them as sensitive.